ChangeLog

```diff
@@ -1,3 +1,37 @@
+What's new in version 3.2.0
+
+* Support for displaying multiple tabs in the user interface
+* Set correct default sorting direction (defaultSortDesc)
+* Improvements to process list tree sorting
```

Comment:
Which PR was this? I don't think this relates to #939.

Reply:
Ah, this was in reference to the series of commits from Denis Lisov around December 13, 16, 19 and again on Jan 7th.

Reply:
Ah, those hashtable fixes …

The current wording may be confusing with the still pending sorting/removal performance fixes.

Maybe reword?
Improve performance for process lookup and update

Reply:
Done, thanks @BenBE